packages/backend/src/utils/npm.ts

```diff
@@ -51,8 +48,14 @@ class Command {
     return _.isEmpty(customInstallationPath) ? "-g" : `--prefix ${customInstallationPath}`;
   }
 
-  private async execCommand(arg: string): Promise<any> {
-    return promisify(exec)(arg);
+  private async execCommand(arg: string): Promise<string> {
+    const result: ExecResult = await promisify(exec)(arg);
+    if (!_.isEmpty(result.stderr)) {
+      // no need to throw error, because stderr usually contains warnings
+      getConsoleWarnLogger().warn(result.stderr);
+    }
+
+    return result.stdout;
```

Comment:
in Window exec can added some chars ('\n' or etc.)  to its stdout result

Reply:
that is why when I use result of execCommand I do _.trim()

Reply:
Done